lib/main.js

```diff
@@ -31,14 +31,30 @@ class PythonLanguageClient extends AutoLanguageClient {
   async startServerProcess(projectPath) {
     await new Promise(resolve => atom.whenShellEnvironmentLoaded(resolve));
 
-    var venvPromise = new Directory(projectPath).getSubdirectory('bin')
-                                                .getFile('python')
-                                                .exists();
+    var venv = false;
+    var venvPath = "";
+
+    new Directory(projectPath)
+      .getEntriesSync()
```

Comment:
Could you use the asynchronous method `getEntries()` because `getEntriesSync()` can block the UI if it takes too long?

Personally I'd wrap it in a Promise to be able to use it with `async`/`await` but you can also use native callbacks. What ever you prefer.

Reply:
I just updated.